src/app/static/src/app/store/modelCalibrate/actions.ts

```diff
@@ -75,13 +76,19 @@ export const actions: ActionTree<ModelCalibrateState, RootState> & ModelCalibrat
 
                 if (data && data.plottingMetadata && data.plottingMetadata.barchart.defaults) {
                     const defaults = data.plottingMetadata.barchart.defaults;
+                    const unfrozenDefaultOptions= Object.keys(defaults.selected_filter_options)
```

Comment:
So is it only the `response.data` object that we want frozen here? In which case I agree with Lekan that simply removing the `freezeResponse` call and doing `const data = freeze(response.data)` would be more parsimonious.